po/fr.po

```diff
@@ -1,5 +1,723 @@
+
+#: src/Dialogs/SettingsDialog.vala:100
+msgid "ToolBar Style"
+msgstr "Style de la barre d'outils"
+
+#: src/Dialogs/SettingsDialog.vala:102
+msgid "Show Button Labels:"
+msgstr "Afficher l'étiquette des boutons :"
+
+#: src/Dialogs/SettingsDialog.vala:106
+msgid "Use Symbolic Icons:"
+msgstr "Utiliser les icônes symboliques :"
+
+#: src/Dialogs/SettingsDialog.vala:129
+msgid "Default Colors"
+msgstr "Couleurs par défaut"
+
+#: src/Dialogs/SettingsDialog.vala:131
+msgid "Define the default style used when creating a new shape."
+msgstr ""
+"Définissez le style par défaut utilisé lors de la création d'une nouvelle "
+"forme."
+
+#: src/Dialogs/SettingsDialog.vala:136
+msgid "Fill Color:"
+msgstr "Couleur de remplissage :"
+
+#: src/Dialogs/SettingsDialog.vala:157
+msgid "Enable Border Style:"
+msgstr "Activer le style de bordure :"
+
+#: src/Dialogs/SettingsDialog.vala:160
+msgid "Border Color:"
+msgstr "Couleur de la bordure :"
+
+#: src/Dialogs/SettingsDialog.vala:181
+msgid "Border Width:"
+msgstr "Largeur de la bordure :"
+
+#: src/Dialogs/SettingsDialog.vala:209
+msgid "The Linux Design Tool"
+msgstr "L'outil de design Linux"
+
+#: src/Dialogs/SettingsDialog.vala:226
+msgid "Thanks to our awesome supporters!"
+msgstr "Merci à tous ceux qui nous soutiennent !"
+
+#: src/Dialogs/SettingsDialog.vala:231
+msgid "View the list of supporters"
+msgstr "Voir la liste des personnes qui soutiennent le projet"
+
+#: src/Dialogs/ShortcutsDialog.vala:46
+msgid "New window:"
+msgstr "Nouvelle fenêtre :"
+
+#: src/Dialogs/ShortcutsDialog.vala:48
+msgid "Preferences:"
+msgstr "Préférences :"
+
+#: src/Dialogs/ShortcutsDialog.vala:50
+msgid "Quit:"
+msgstr "Quitter :"
+
+#: src/Dialogs/ShortcutsDialog.vala:52
+msgid "Presentation mode:"
+msgstr "Mode présentation :"
+
+#: src/Dialogs/ShortcutsDialog.vala:55
+msgid "File"
+msgstr "Ficher"
+
+#: src/Dialogs/ShortcutsDialog.vala:56
+msgid "Open:"
+msgstr "Ouvrir :"
+
+#: src/Dialogs/ShortcutsDialog.vala:58
+msgid "Save:"
+msgstr "Enregistrer :"
+
+#: src/Dialogs/ShortcutsDialog.vala:60
+msgid "Save as:"
+msgstr "Enregistrer sous :"
+
+#: src/Dialogs/ShortcutsDialog.vala:64
+msgid "Export artboards:"
+msgstr "Exporter les plans de travail :"
+
+#: src/Dialogs/ShortcutsDialog.vala:66
+msgid "Export selection:"
+msgstr "Exporter la sélection :"
+
+#: src/Dialogs/ShortcutsDialog.vala:68
+msgid "Highlight area to export:"
+msgstr "Zone en surbrillance à exporter :"
+
+#: src/Dialogs/ShortcutsDialog.vala:77
+msgid "Canvas"
+msgstr "Canevas"
+
+#: src/Dialogs/ShortcutsDialog.vala:78
+msgid "Zoom in:"
+msgstr "Zoomer :"
+
+#: src/Dialogs/ShortcutsDialog.vala:80
+msgid "Zoom out:"
+msgstr "Dézoomer :"
+
+#: src/Dialogs/ShortcutsDialog.vala:82
+msgid "Zoom reset:"
+msgstr "Mise à zéro du zoom :"
+
+#: src/Dialogs/ShortcutsDialog.vala:85
+msgid "Item creation"
+msgstr "Création d'élément"
+
+#: src/Dialogs/ShortcutsDialog.vala:86
+msgid "Artboard:"
+msgstr "Plan de travail :"
```

Comment:
I wonder if `espace de travail` would be a better translation of `artboard`, as `plan de travail` refers mainly to a physical object, but I might be wrong.

Reply:
I didn't like the translation "plan de travail", but I didn't have a better one. "espace de travail" seems better to me. But if anyone has better, please feel free to say so.